app/src/main/java/edu/uc/app/parkfinder/RetrofitClientInstance.kt

```diff
@@ -6,12 +6,12 @@ import retrofit2.converter.gson.GsonConverterFactory
 object RetrofitClientInstance {
 
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://pkgstore.datahub.io"
+    private const val BASE_URL = "https://pkgstore.datahub.io"
 
     val retrofitInstance: Retrofit?
         get() {
             if (retrofit == null) {
-                retrofit = retrofit2.Retrofit.Builder()
```

Comment:
cleared redundant retrofit2

Reply:
This duplicates a previous code review.  When that occurs, first in wins.